conftest.py

```diff
@@ -83,3 +89,59 @@ def test_db(tasks):
     managers = tasks[2]
     task = tasks[3]
     return teams, employees, managers, task
+
+
+@pytest.fixture
+def valid_teams():
+    return [Team.objects.create(name=f"Team{index}", description="Best team ever") for index in range(1, 4)]
```

Comment:
maybe it will be better to make unique description for each team 
```suggestion
    return [Team.objects.create(name=f"Team{index}", description=f"Best team{index} ever") for index in range(1, 4)]
```

Reply:
Not mine, I just moved it there

Reply:
even though it is not your code, I think when it's not core changes only cosmetics it is no big deal to change it